Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -554,4 +524,23 @@ class SettingsTableSectionHeaderFooterView: UITableViewHeaderFooterView {
         titleLabel.text = nil
         titleAlignment = .Bottom
     }
+
+    private func remakeTitleAlignmentConstraints() {
+        switch titleAlignment {
+        case .Top:
+            titleLabel.snp_remakeConstraints { make in
+                make.left.equalTo(self).offset(SettingsTableSectionHeaderFooterViewUX.titleHorizontalPadding)
+                make.right.lessThanOrEqualTo(self).offset(-SettingsTableSectionHeaderFooterViewUX.titleHorizontalPadding)
+                make.top.equalTo(self).offset(SettingsTableSectionHeaderFooterViewUX.titleVerticalPadding)
+                make.bottom.equalTo(self).offset(-SettingsTableSectionHeaderFooterViewUX.titleVerticalLongPadding)
+            }
```

Comment:
SnapKit supports `.offset(UIEdgeInsets)`, and also `.inset(UIEdgeInsets)` (haven't looked at the difference carefully). Can we turn all of these into a one-liner using one of those?